bindings/rascal/representations/spherical_expansion.py

```diff
@@ -150,3 +150,12 @@ def get_num_coefficients(self):
         """
         return (self.hypers['n_species'] * self.hypers['max_radial']
                 * (self.hypers['max_angular'] + 1)**2)
+
+    def get_keys(self, species):
+        """
+        return the proper list of keys used to build the representation
+        """
+        keys = []
+        for sp in species:
```

Comment:
I am not sure to understand

Reply:
Sorry, phone api isn't great, accidentally resolved conversation. 

If someone calls get_dense_feature_matrix on the same list of species, just in different order, will they get the same soap matrix?

Reply:
> If someone calls get_dense_feature_matrix on the same list of species, just in different order, will they get the same soap matrix?

Yes it will be the same because the keys from python are moved to a std::set (ordered container with unique entries).